<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scan_control</class>
 <widget class="QWidget" name="Scan_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ПЭТ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_device"/>
     </item>
     <item>
      <widget class="QToolButton" name="btn_refresh">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string>Пуск</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tw_state_pet">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_refresh_state">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_auto_refresh_state">
       <property name="text">
        <string>автоматически</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_interval_auto_state"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>мс</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_collapsed_all">
       <property name="text">
        <string>Свернуть все</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_expanding_all">
       <property name="text">
        <string>Развернуть все</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
